examples/petset/mongodb-petset-persistent.yaml

```diff
@@ -69,6 +69,10 @@ parameters:
     displayName: "Memory Limit"
     description: "Maximum amount of memory the container can use."
     value: "512Mi"
+  - name: TIMEZONE
```

Comment:
@git001 Please add empty line before - to "separate" this new parameter. (to make it consistent with other parameters)